demo/src/parse5-html-rewriting-stream.d.ts

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
Hopefully https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28660 will be published soon.